cli/cmd/inject.go

```diff
@@ -89,9 +88,14 @@ sub-folders, or coming from stdin.`,
 			if err != nil {
 				return err
 			}
+			overrideAnnotations := map[string]string{}
+			options.overrideConfigs(configs, overrideAnnotations)
 
-			options.overrideConfigs(configs)
-			exitCode := uninjectAndInject(in, stderr, stdout, configs)
+			transformer := &resourceTransformerInject{
+				configs:             configs,
+				overrideAnnotations: overrideAnnotations,
+			}
```

Comment:
How about defining this earlier, and have `overridedConfigs()` receive it as its sole argument?

Reply:
I prefer to keep the arguments of `overrideConfigs()` to config data. A function that reads `overrideConfigs(someKindOfTransformer)` sounds like we are overriding the transformer's config, which it isn't because the transformer has no configs of its own, and it's more a YAML-parsing receiver, than a config receiver.